src/ic-xrc-types/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub struct ExchangeRateMetadata {
 pub struct ExchangeRate {
     /// The base asset.
     pub base_asset: Asset,
-    /// The quote asset.
+    /// The quoted asset.
```

Comment:
Same here.

Reply:
Regardless of "quote" vs. "quoted", putting a comment `// The foo` on a field called `foo`, doesn't add any value IMO. Can we just remove this?

Reply:
> doesn't add any value

👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

FWIW, it was already like that.

I'd prefer to just delete it, but I don't own this, and deleting would be outside the original intent of this PR -> I do as Thomas says 🙇

Reply:
:laughing: 
I agree with @dskloetd (and you). Feel free to delete these comment lines as part of this PR!